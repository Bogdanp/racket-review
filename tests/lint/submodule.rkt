#lang racket/base

(module unsafe racket/base
  (provide (all-defined-out))
  (define x 42))

(module foo racket/base
  (provide y)
  (define x 42)
  (define (f x)
    (println x)))

(module bar racket/base
  (require racket/string)
  (provide non-empty-string?))

(provide x)
