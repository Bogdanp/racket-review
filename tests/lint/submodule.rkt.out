/submodule.rkt:8:12:warning:identifier 'y' provided but not defined
/submodule.rkt:10:14:warning:identifier 'x' shadows an earlier binding
/submodule.rkt:15:12:warning:identifier 'non-empty-string?' provided but not defined
/submodule.rkt:17:10:warning:identifier 'x' provided but not defined
